script/lib/connect.py

```diff
@@ -1,22 +0,0 @@
-import base64
```

Comment:
Also safe to remove; this was only being used by `script/lib/omaha.py` which was removed with https://github.com/brave/brave-core/pull/8543